### ISSUE TITLE
--debug option should pass the debug flag to the translator

### ISSUE THREAD

**Issue by clementval** [C2SM-RCM/claw-compiler#24]
Currently, to have debug information from the translator, we have to pass the debug flag through the transflator specififc flag like this 

```bash
clawfc --Wx-d
```

It would be nice to have this flag passed automatically when the "main" debug flag is on

```bash
clawfc --debug
```
